parachains/runtimes/collectives/collectives-polkadot/src/constants.rs

```diff
@@ -13,6 +13,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+pub mod account {
+	use frame_support::PalletId;
+	use sp_runtime::AccountId32;
+
+	/// realy chain treasury pallet id, used to convert into AccountId
```

Comment:
Don't know how I missed this one.
```suggestion
	/// Relay Chain treasury pallet id, used to convert into AccountId
```